qa/tasks/cephfs/test_damage.py

```diff
@@ -662,3 +662,33 @@ def test_dentry_first_precommit(self):
         self.mount_a.mount_wait()
         self.mount_a.run_shell_payload("stat a/ && find a/")
         self.fs.flush()
+
+    def test_missing_dirfrag_create(self):
+        """
+        That the MDS won't let you manipulate dentries in a missing dirfrag.
+        """
+
+        self.mount_a.run_shell_payload("mkdir -p dir_x/dir_xx/dir_xxx/")
+        self.mount_a.run_shell_payload("touch dir_x/dir_xx/dir_xxx/file_y")
```

Comment:
nit: I recommend using `self.mount_a.touch()`

Reply:
In the interest of keeping this settled, I'll leave this for now.